common/src/main/java/io/github/gaming32/bingo/BingoCommand.java

```diff
@@ -129,6 +129,10 @@ public class BingoCommand {
     private static final CommandSwitch<Long> SEED = CommandSwitch
         .argument("--seed", LongArgumentType.longArg())
         .build(RandomSupport::generateUniqueSeed);
+    private static final CommandSwitch<Integer> TIME = CommandSwitch
+        .argument("--time", IntegerArgumentType.integer())
```

Comment:
Suggestion: what about using `TimeArgument` instead, like `--auto-forfeit-time` does? That allows you to specify a time unit suffix (e.g. `2d` for 2 Minecraft days, `300s` for 5 minutes), the downside is that there's no way to specify minutes.